### PR TITLE
NE: 104th Session is working for legislators

### PR DESCRIPTION
**Pull request by mileswwatkins**
Bills (and thus votes) aren't up, and committee appointments haven't been updated.

### CODE REVIEW

File: openstates/ne/legislators.py [hunk 1]
```diff
@@ -30,12 +30,18 @@ def scrape(self, term, chambers):
                     phone = phone[1] + ' ' + phone[2]
                 else:
                     phone = None
-                email = page.xpath('//div[@id="sidebar"]/ul[1]/li[7]/a')[0].text or ''
+                mailto = page.xpath('//div[@id="sidebar"]/ul[1]/li[contains(text(), "Email:")]/a/@href')[0]
+                email = mailto[7:]
+
+                photo_url = \
+                        "http://www.nebraskalegislature.gov/media/images/blogs/dist%d02.jpg" \
```
Comment (calebcassel) [inline openstates/ne/legislators.py]: These images don't exist. It looks like it works if you remove the "02" after %d.

Comment (jamesturk) [inline openstates/ne/legislators.py]: I'd guess that is supposed to be %02d as the images are at
http://www.nebraskalegislature.gov/media/images/blogs/dist01.jpg for single
digits

On Wed, Jan 7, 2015 at 4:32 PM, Caleb Cassel <notifications@github.com>
wrote:

> In openstates/ne/legislators.py
> <https://github.com/sunlightlabs/openstates/pull/499#discussion-diff-22617865>
> :
>
> > @@ -30,12 +30,18 @@ def scrape(self, term, chambers):
> >                      phone = phone[1] + ' ' + phone[2]
> >                  else:
> >                      phone = None
> > -                email = page.xpath('//div[@id="sidebar"]/ul[1]/li[7]/a')[0].text or ''
> > +                mailto = page.xpath('//div[@id="sidebar"]/ul[1]/li[contains(text(), "Email:")]/a/@href')[0]
> > +                email = mailto[7:]
> > +
> > +                photo_url = \
> > +                        "http://www.nebraskalegislature.gov/media/images/blogs/dist%d02.jpg" \
>
> These images don't exist. It looks like it works if you remove the "02"
> after %d.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sunlightlabs/openstates/pull/499/files#r22617865>.
>



-- 

James Turk
Labs Director | Sunlight Foundation <http://www.sunlightfoundation.com/>
202-558-8723 | @JamesTurk <https://twitter.com/jamesturk>

[image: Sunlight Foundation] <http://sunlightfoundation.com/>

Comment (mileswwatkins) [inline openstates/ne/legislators.py]: Argh, yes.

Comment (mileswwatkins) [inline openstates/ne/legislators.py]: Fixed.

Comment (paultag) [inline openstates/ne/legislators.py]: Sweet, looks like @mileswwatkins updated this -- thanks so much @calebcassel!

**Pull request merged**